README.md

```diff
@@ -33,7 +35,7 @@ syntax for GraphQL schema definition.
 
 ```groovy
 dependencies {
-  compile "io.github.graphql-java:graphql-java-annotations:6.2"
+  compile "io.github.graphql-java:graphql-java-annotations:XX"
```

Comment:
why

Reply:
because when people use a library, they just want to copy paste the dependency to their project, they don't want to search the latest version in maven